src/v2/Apps/Search/Components/NavigationTabs.tsx

```diff
@@ -57,7 +58,7 @@ export class NavigationTabs extends React.Component<Props> {
     const tabName = text.replace(/[0-9]/g, "").trim()
     return (
       <RouteTab
-        to={to}
+        to={to.replace(/\s/g, "_")}
```

Comment:
Why was this change necessary?

Reply:
Removes whitespace from the tab (so /artist series -> /artist_series).